torch/xpu/__init__.py

```diff
@@ -448,6 +435,7 @@ def _get_rng_state_offset(device: Union[int, str, torch.device] = "xpu") -> int:
     return default_generator.get_offset()
 
 
+from .memory import *  # noqa: F403
```

Comment:
nit: explicit imports are better for typing and readability. Would be nice to fix that going forward and remove the noqa

Reply:
Thanks very much. I removed `noqa` in this PR.